features/steps/tabs.py

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+
+"""
+Step implementations for paragraph-related features
+"""
+
+from behave import given, then, when
+
+from docx import Document
+from docx.text.tabstop import TabStop
+from docx.shared import Inches
+
+from helpers import test_docx
+
+
+# given ===================================================
+@given('a tab_stops having {count} tab stops')
+def given_a_tab_stops_having_count_tab_stops(context, count):
+    count_int = {
+        '0': 0,
+        '3': 1
+    }[count]
+    document = Document(test_docx('tab-stops'))
+    paragraph_format = document.paragraphs[count_int].paragraph_format
+    context.tab_stops = paragraph_format.tab_stops
+
+
+# when ====================================================
+
+@when('I add a tab_stop')
+def when_i_add_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops.add_tab()
+    tab.position = Inches(1.75)
+
+
+@when('I delete a tab_stop')
+def when_i_delete_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops[1]
+    tab.delete()
```

Comment:
Okay, I did a little more research on the `del` statement and I definitely love it a lot less now. I was under the mistaken impression that `del` just directly called the `__del__()` method of the object, but that's not how it works it turns out :)

I'd say let's go with .remove() instead (analogous to the list method of the same name):

```python
tab_stops.remove(tab_stops[1])
```

This may actually be more useful, because it takes a TabStop object reference rather than an index, so you could use it in a for loop for example:

```python
for tab_stop in tab_stops:
    if i_dont_like_this(tab_stop):
        tab_stops.remove(tab_stop)
```